docs/releases/changelog-dev.md

```diff
@@ -4,6 +4,9 @@
 
 ### Improvements
 
+- The method `CalibrationNode._execute_notebook()` now changes the working directory to the notebook directory before the execution and restores the previous one after the papermill execution. It allows the notebooks now to use relative paths.
```

Comment:
Update the changelog, into saying that the full class, since initialization will now work with absolute paths?

Reply:
Yep makes sense!